test/e2e/fixtures/e2e_suite.go

```diff
@@ -74,9 +76,27 @@ func (s *E2ESuite) BeforeTest(string, string) {
 	s.DeleteResources()
 }
 
+func (s *E2ESuite) AfterTest(suiteName, testName string) {
+	if s.T().Failed() { // by default, we don't get good logging at test end
```

Comment:
The lack of logging is definitely a pain. I had to manually modify the scripts to print out the logs for failed workflows in the past.